docs/docs/schemas.md

```diff
@@ -34,9 +34,16 @@ Iceberg tables support the following types:
 | **`time`**         | Time of day without date, timezone                                       | Stored as microseconds                           |
 | **`timestamp`**    | Timestamp without timezone                                               | Stored as microseconds                           |
 | **`timestamptz`**  | Timestamp with timezone                                                  | Stored as microseconds                           |
+| **`timestamp_ns`**     | Timestamp without timezone, nanosecond precision                         | Stored as nanoseconds; added in v3               |
+| **`timestamptz_ns`**   | Timestamp with timezone, nanosecond precision                            | Stored as nanoseconds; added in v3               |
 | **`string`**       | Arbitrary-length character sequences                                     | Encoded with UTF-8                               |
+| **`uuid`**             | Universally unique identifiers                                           | Should use 16-byte fixed                         |
 | **`fixed(L)`**     | Fixed-length byte array of length L                                      |                                                  |
 | **`binary`**       | Arbitrary-length byte array                                              |                                                  |
+| **`variant`**          | Semi-structured data (JSON-like)                                         | Added in v3; requires Spark 4.0+                 |
+| **`geometry(C)`**      | Geospatial features with CRS parameter                                   | Planar calculations; added in v3                 |
```

Comment:
not sure "calculations" is the right word here, I don't know the Geo Space well enough but I would have guessed maybe kernel? Or Topography